.github/workflows/build-nym-vpn-app-windows.yml

```diff
@@ -231,6 +243,7 @@ jobs:
 
       - name: Build tauri desktop client
         working-directory: nym-vpn-app/src-tauri
+        if: ${{ !inputs.sign }}
```

Comment:
same as before, let's remove this step duplication and merge it with the one bellow, using gh expression based on `input.sign` value to set the relevant env vars